mysql-test/suite/galera/t/pxc_strict_mode.test

```diff
@@ -0,0 +1,1189 @@
+
+#
+# Test different options with different variation of pxc-strict-mode.
+#
+
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+
+
+#-------------------------------------------------------------------------------
+#
+# create test-bed
+#
+# let's cache the default value first.
+--connection node_1
+--echo #node-1
+--let $pxc_strict_mode_saved1 = `SELECT @@pxc_strict_mode`
+--let restartdir= $MYSQLTEST_VARDIR/tmp/restartdir
+--mkdir $restartdir
+
+--connection node_1
```

Comment:
Not sure I understood this comment exactly as to which part it is referring too. BTW we no more need to set DISABLED mode as default for mtr is DISABLED.